src/librustc_mir/util/liveness.rs

```diff
@@ -388,12 +393,12 @@ fn block<'tcx>(mode: LivenessMode, b: &BasicBlockData<'tcx>, locals: usize) -> D
     visitor.defs_uses
 }
 
-pub fn dump_mir<'a, 'tcx>(
+pub fn dump_mir<'a, 'tcx, V: Idx>(
```

Comment:
this code will also need a `&impl LiveVariableMap<LiveVar = V>` to be given to it

Reply:
it can use that in the printouts below to recover a `Local`